src/core/packages/saved-objects/api-server-internal/src/lib/apis/update.ts

```diff
@@ -112,7 +112,14 @@ export const executeUpdate = async <T>(
   const existingNamespaces = preflightDocNSResult.savedObjectNamespaces ?? [];
   const authorizationResult = await securityExtension?.authorizeUpdate({
     namespace,
-    object: { type, id, existingNamespaces },
+    object: {
```

Comment:
Not very familiar with this extension, so just a q FMI: but you don't need `objectNamespace` passed in here?